client/rpcserver/handlers.go

```diff
@@ -589,8 +589,11 @@ Registration is complete after the fee transaction has been confirmed.`,
             "fundConf" (int): The number of confirmations needed before coins
 	      can be traded.
           },...
-        },
-        "feePending" (bool): Whether the dex fee is pending.
+		},
+		"confsrequired": (int) The number of confirmations needed for the 
+		registration fee payment
+		"confs" (int/null): The current number of confirmations for the registration
+		fee payment or null when the payment is confirmed
```

Comment:
Consider using a `json:"confs,omitempty"` here. Maybe say something like `... only present during the registration process.`